packages/menu/src/Menu.ts

```diff
@@ -678,7 +665,7 @@ export class Menu extends SizedMixin(SpectrumElement, { noDefaultSize: true }) {
             if (lastFocusedItem?.hasSubmenu) {
                 //open submenu and set focus
                 event.stopPropagation();
-                lastFocusedItem.openOverlay();
+                lastFocusedItem.openOverlay(true);
```

Comment:
Added to distinguish between deliberate events, like clicking and keyboard opening and casual events like hovering. We don't want focus automatically set or the focus indicator would be one step ahead of mouse hover.

Reply:
great point!